larq_compute_engine/tflite/kernels/lce_ops_register.h

```diff
@@ -2,24 +2,36 @@
 #define COMPUTE_ENGINE_TFLITE_KERNELS_LCE_OPS_REGISTER_H_
 
 #include "tensorflow/lite/context.h"
+//#include "tensorflow/lite/stderr_reporter.h"
```

Comment:
In case you missed it, this one can be removed. I think the `TFLITE_LOG(WARN)` solution is good :+1:

Reply:
thanks a lot 🤙. It worked